jwql/website/apps/jwql/static/filesystem

```diff
@@ -0,0 +1 @@
+/grp/jwst/ins/jwql/filesystem/
```

Comment:
@tnking97 I don't think these `filesystem`, `preview_images`, `thumbnails`, and `jwql/website/db.sqlite3` files should be committed to the repository, could you remove them from this PR?